src/client/buttons/DislikeButton/DislikeButton.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import IconButton from '../IconButton';
+
+const DislikeButton = (props) => {
+  const { className, onClick } = props;
+
+  return (
+    <IconButton
+      icon=""
+      className={`dislike-button ${className}`}
```

Comment:
Usually we don't pass in `classNames` as props, but i can see why you did. (i guess you're trying to make the component reusable while at the same time you need it to be specific based on where it is rendered) 

One way i'd do it is pass in a `parent` prop to indicate where `DislikeButton` is being rendered from. Then within `DislikeButton` component, set up a `switch` statement to add the associated className.

```js
// if being rendered from Swipe component
...
<DislikeButton parent="swipe" onClick={onClick} />
...
```

```js
// in DislikeButton 
const {parent} = props;
let className="dislike-button ";
switch(parent) {
   case 'swipe': 
     className += "swipe ";
     break;
   case 'another-component':
     className += "another-className ";
     break;
 ...
```

but it's okay if you want to stick to current method. it's fine too.